src/main/asciidoc/index.adoc

```diff
@@ -149,6 +149,16 @@ Generally-speaking, we respect the JUnit extension scoping rules, but here are t
 3. Injecting in a `@BeforeEach` with no parent context or previous `@BeforeAll` injection creates a new instance shared with the corresponding test and `AfterEach` method(s).
 4. When no instance exists before running a test method, an instance is created for that test (and only for that test).
 
+==== Configuring of `Vertx` objects
```

Comment:
```suggestion
==== Configuring `Vertx` instances
```

Reply:
The following paragraph (original documentation) uses "`Vertx` objects", so would that need change too?